src/assets/stylesheets/_extensions.scss

```diff
@@ -1,3 +1,8 @@
 .govuk-footer {
   border-top: govuk-spacing(2) solid govuk-colour('blue');
 }
+
+.extra-letter-spacing {
```

Comment:
Might be worth turning this into override classes, following the design system pattern. I mean something like:
```css
.govuk-\!-tabular-font {
  @include govuk-font($size: false, $tabular: true);
}

.govuk-\!-letter-spacing-two-thirds {
  letter-spacing: .06em;
}
```

Reply:
I’ve adopted this suggestion in 08aaf29